src/main/java/marquez/common/models/JobName.java

```diff
@@ -25,4 +25,8 @@
 @ToString
 public final class JobName {
   @Getter @NonNull private final String value;
+
+  public String getVal() {
```

Comment:
A getter method will be generated by lombok with [`@Getter`](https://projectlombok.org/features/GetterSetter).  So, we can remove `getVal()`. Please see [`JobNameTest`](https://github.com/MarquezProject/marquez/blob/master/src/test/java/marquez/common/models/JobNameTest.java) for example usage.

Reply:
Question: How can I access the @Getter in jobResource.java? 
![marquezerror](https://user-images.githubusercontent.com/27025266/53878904-14270500-3fdb-11e9-818e-7a057d198d9b.png)
I replaced all the job.getVal() with jobName(I know it doesn't make sense because I am trying to switch a class with a string) I get errors. @wslulciuc